cpu/stm32/periph/flashpage.c

```diff
@@ -253,3 +319,54 @@ void flashpage_write(void *target_addr, const void *data, size_t len)
     }
 #endif
 }
+
+#if !(defined(CPU_FAM_STM32F2) || defined(CPU_FAM_STM32F4) || \
+      defined(CPU_FAM_STM32F7))
+void flashpage_write_page(unsigned page, const void *data)
+{
+    flashpage_erase(page);
+
+    /* WRITE sequence */
+    if (data != NULL) {
+        flashpage_write(flashpage_addr(page), data, FLASHPAGE_SIZE);
+    }
+}
+#endif
+
+#if defined(CPU_FAM_STM32F2) || defined(CPU_FAM_STM32F4) || \
+    defined(CPU_FAM_STM32F7)
+size_t flashpage_size(unsigned page)
+{
+    if (page < 4) {
+        return 16 * 1024;
+    }
+    else if (page == 4) {
+        return 64 * 1024;
+    }
+    else {
+        return 128 * 1024;
+    }
```

Comment:
In think we have nice macros for this now if you want to use them `KiB(x)`

Reply:
Fixed!